rust/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
should this be `rust/target` ?

Reply:
no, it's relative to the directory containing the gitignore -- it felt vaguely correct to give the rust subproject its own gitignore since different build system, IDE, etc will be used there

Reply:
oh woops I thought this was in the root .gitigore, 🙃